CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ the same as the last release note version.
 * Enhanced the dependency message shown when running **lint**.
 * Fixed an issue where **update-release-notes** didn't update the currentVersion in pack_metadata
 * Improved the logging in **test-content** for helping catch typos in external playbook configuration.
+* Fixed an issue where **update-xsoar-config-file** does not insert properly the marketplace packs to the list.
```

Comment:
```suggestion
* Fixed an issue where **update-xsoar-config-file** did not properly insert the marketplace packs to the list.
```

What packs were not inserted? Why were they not inserted?

Reply:
done